account_asset_management/models/account_asset_line.py

```diff
@@ -35,9 +35,7 @@ class AccountAssetLine(models.Model):
         string="Depreciation Base",
         currency_field="company_currency_id",
     )
-    amount = fields.Monetary(
-        string="Amount", required=True, currency_field="company_currency_id"
-    )
+    amount = fields.Monetary(required=True, currency_field="company_currency_id")
```

Comment:
Please change also the currency on this model to `currency_id` and remove all the `currency_field` references. In fact, the related field should be to the asset currency, not to the company one (although right now is the same).

Reply:
I did that in my commit [[REF] account_asset_management: the name of the company_currency_id](https://github.com/OCA/account-financial-tools/pull/1479/commits/73b3200189d15bcdaa97e81def78642b42a671d6). didn't know if I should apply fixup with commit MIG.